Monika After Story/game/script-topics.rpy

```diff
@@ -6937,6 +6937,308 @@ label monika_citizenship:
     m 2eka "But I want to make you happy, so...I'm going to do everything I can to keep bettering myself while I'm stuck here!"
     m 1eka "Thank you for listening to me vent, [player]."
     return
+    
+#Local variable to check if someone has skipped
+default persistent._mas_pm_monika_listened = None
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_graduation",category=['school'],prompt="Graduation",random=True))
+
+label monika_graduation:
+    m 2ekc "You know, [player], one thing I really wish I got to experience is my high school graduation."
+    m "Pretty much my entire life so far has revolved around school."
+    m 4dkd "All those countless hours of studying, all the afterschool activities and clubs..."
+    m 4ekc "And in the end, after all that hard work, I never got to experience the fulfillment of actually graduating."
+    m 2dkd "Never got to walk across the stage and receive my diploma."
+    m "Never got to hear my name being announced and all my friends cheering."
+    m 2ekc "...It kinda feels like it was all for nothing."
+    m 2esd "I know all the things I learned along the way are what's really important."
+    m 2dkc "But it still feels like I missed out on something special."
+    m "..."
+    
+    #Went through and timed out on the menu twice
+    if not persistent._mas_pm_monika_listened is False:
```

Comment:
```suggestion
    if persistent._mas_pm_monika_listened is False:
```

Reply:
`persistent._mas_pm_monika_listened` returns None but both `not persistent._mas_pm_monika_listened is False` and `not persistent._mas_pm_monika_listened` return True so even if it is your first time seeing this event you never reach the else and `monika_grad_speech_call` remains locked.